File: client/src/models/todo.ts
class Todo {
    id: number;
    title: string;
    context: string;
    limit_date: string;
    created_at: string;
    updated_at: string;

    constructor(id: number, title: string, context: string, limit_date: string, create_at: string, updated_at: string) {
        this.id = id;
        this.title = title;
        this.context = context;
        this.limit_date = limit_date;
        this.created_at = create_at;
        this.updated_at = updated_at;
    }

}

File: api/model/todo.go
package model

import (
	"time"
)

type Todo struct {
	ID        int       `json:"id"         db:"id"`
	Title     string    `json:"title"      db:"title"`
	Context   string    `json:"context"    db:"context"`
	LimitDate time.Time `json:"limit_date" db:"limitDate"`
	CreatedAt time.Time `json:"created_at" db:"createdAt"`
	UpdatedAt time.Time `json:"updated_at" db:"updatedAt"`
}